systemaudioplatform/systemaudioplatform.cpp

```diff
@@ -0,0 +1,49 @@
+#include <cmath>
+#include "systemaudioplatform.h"
+
+void systemAudioInitialize()
+{
+    //NA
+}
+
+void systemAudioDeinitialize()
+{
+    //NA
+}
+
+void systemAudioChangePrimaryVol(MixGain gain, int volume)
+{
+    //NA
+}
+
+bool systemAudioGeneratePipeline(GstElement *pipeline, GstElement *source,
+                                 GstElement *capsfilter, GstElement **audioSink,
+                                 GstElement **audioVolume, AudioType type,
+                                 PlayMode mode, SourceType sourceType,
+                                 bool smartVolumeEnable)
+{
+    return true;
```

Comment:
It seems better to me that we return "FALSE" instead of "TRUE".
Since this is a dummy implementation; this also is fine.